ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge1.cfg

```diff
@@ -317,6 +413,56 @@ application traffic recognition
    field-set l4-port TCP-SRC-2
       42
 !
+monitor connectivity
+   interface set ZSTunnel100 Tunnel100
+   interface set ZSTunnel101 Tunnel101
+   interface set ZSTunnel102 Tunnel102
+   interface set ZSTunnel110 Tunnel110
+   interface set ZSTunnel111 Tunnel111
+   interface set ZSTunnel112 Tunnel112
+   !
+   host ZSTunnel100
+      description
+      Zscaler tunnel
+      local-interfaces ZSTunnel100 address-only
+      ip 172.25.42.16/31
+      url http://TODO
+   !
+   host ZSTunnel101
+      description
+      Zscaler tunnel
+      local-interfaces ZSTunnel101 address-only
+      ip 172.25.42.16/31
+      url http://TODO
+   !
+   host ZSTunnel102
+      description
+      Zscaler tunnel
+      local-interfaces ZSTunnel102 address-only
+      ip 172.25.42.16/31
+      url http://TODO
```

Comment:
may require a host entry for the `http://gateway.zscalerbeta.net/vpntest`

Reply:
We can just add a limitation that you must have name server configured.